api/api_full.go

```diff
@@ -172,6 +172,10 @@ type FullNode interface {
 	// If oldmsgskip is set, messages from before the requested roots are also not included.
 	ChainExport(ctx context.Context, nroots abi.ChainEpoch, oldmsgskip bool, tsk types.TipSetKey) (<-chan []byte, error) //perm:read
 
+	ChainExportRange(ctx context.Context, head, tail types.TipSetKey, cfg ChainExportConfig) (<-chan []byte, error) //perm:read
+
+	ChainExportRangeInternal(ctx context.Context, head, tail types.TipSetKey, cfg ChainExportConfig) error //perm:read
```

Comment:
This one writes a lot of data locally, definitely higher perm that read; I'd say it should be `admin` because it's easy enough to abuse to bring the node down

Reply:
https://github.com/filecoin-project/lotus/pull/10145/commits/71b478e38fb835ef4572b0157feb0e8b4927559d